docs/source/tutorial.rst

```diff
@@ -623,6 +623,7 @@ Useful grep commands
 
     $ grep "WARNING\|ERROR" output/*log
     $ grep -n "srun\|slurmstepd\|Traceback" output/*.e
+    $ grep "\bpercent\b" output/stats_summary.json | awk '{print $2}' | sort -n
```

Comment:
Would you mind adding something like the line below so that people know what this is? Most people will not know what this is.
```
# Check max memory usage per node.
```

Reply:
of course! updated.